backend/app/utils.py

```
@@ -3166,6 +3166,20 @@ def _find_map_json(directory):
+    possible_locations = [
+        directory / "Index.json"
+    ]
+    try:
+        for location in possible_locations:
+            if location.exists():
+                with open(location, 'r') as f:
+                    data = json.load(f)
+                for node in data.get("Nodes", []):
+                    if node.get("NodeType", None) == "map":
+                        return node.get("FileName")
+    except:
```

Ruff (0.15.10)

[error] Do not use bare `except`

(E722)